R/calendar.R

```diff
@@ -9,6 +9,8 @@
 #' @param type Select from \code{'year'}, \code{'month'}, \code{'date'} and \code{'time'}
 #' @param min Minimum allowed value.
 #' @param max Maximum allowed value.
+#' @param start_calendar_id id of the calendar that defines the range start.
+#' @param end_calendar_id id of the calendar that defines the range end.
```

Comment:
I think we should create additional function that uses the original calendar() function and adds new parameters instead of adding the parameters to exported function. It may be confusing, especially that in the examples it seems like the user needs to provide start and end calendar id